captum/optim/models/_image/inception_v1.py

```diff
@@ -315,7 +314,7 @@ def __init__(
         self,
         in_channels: int = 508,
         out_features: int = 1008,
-        activ=ReluLayer,
+        activ=nn.ReLU,
```

Comment:
Suggestion: This should also be type hinted

Reply:
Done!